hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -416,7 +416,9 @@ private boolean renameV1(String srcPath, String dstPath) throws IOException {
     } catch (OMException ome) {
       LOG.error("rename key failed: {}. source:{}, destin:{}",
               ome.getMessage(), srcPath, dstPath);
-      if (OMException.ResultCodes.KEY_ALREADY_EXISTS == ome.getResult()) {
+      if (OMException.ResultCodes.KEY_ALREADY_EXISTS == ome.getResult() ||
```

Comment:
Please add error code to the above log message to improve debugging.  `"error code: {}", ome.getResult()`

Reply:
Done.